batch-oai-harvest/src/main/java/data/unit/writer/OaiIdentifiersWriter.java

```diff
@@ -0,0 +1,30 @@
+package data.unit.writer;
+
+import data.entity.ExecutionRecordExternalIdentifier;
+import data.repositories.ExecutionRecordExternalIdentifierRepository;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.batch.item.Chunk;
+import org.springframework.batch.item.ItemWriter;
+import org.springframework.stereotype.Component;
+
+import java.lang.invoke.MethodHandles;
+
+@Component
+public class OaiIdentifiersWriter implements ItemWriter<ExecutionRecordExternalIdentifier> {
```

Comment:
Use RepositoryItemWriter similar to DefaultRepositoryItemWriter. Makes the code much simpler.